synapse/storage/search.py

```diff
@@ -56,17 +54,6 @@ def __init__(self, db_conn, hs):
             self.EVENT_SEARCH_ORDER_UPDATE_NAME, self._background_reindex_search_order
         )
 
-        # we used to have a background update to turn the GIN index into a
```

Comment:
Is it necessary to remove these? I'd prefer to keep them if they aren't doing any harm. People do upgrade from old versions.

Reply:
OK -- I'll split this into another discussion.